File: laravel-app/app/Models/PollResult.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class PollResult extends Model
{
    protected $fillable = [
        'id',
        'poll_id',
        'is_draw',
        'total_votes'
    ];
    public function winningOption()
    {
        return $this->belongsTo(PollOption::class, 'finalized_option');
    }
}

File: netwinform-app/Models/PollResult.cs
using System;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace netwinform_app.Models
{
    public class PollResult
    {
        [JsonProperty("id")]
        public string Id { get; set; }

        [JsonProperty("poll_id")]
        public string PollId { get; set; }

        [JsonProperty("is_draw")]
        public bool IsDraw { get; set; }

        [JsonProperty("total_votes")]
        public string TotalVotes { get; set; }
    }
}
